shared/src/packages/orders/libs/validation-schemas/orders-update.validation-schema.ts

```diff
@@ -0,0 +1,19 @@
+import joi from 'joi';
+
+import { UserValidationRule } from '~/packages/users/libs/validation-schemas/enums/user-validation-rule.enum.js';
+
+import { OrderStatus } from '../enums/order-status.enum.js';
+import { type OrderUpdateRequestDto } from '../types/order-update-request-dto.type.js';
+
+const orderUpdate = joi.object<OrderUpdateRequestDto, true>({
+  scheduledTime: joi.string().isoDate(),
+  startPoint: joi.string(),
+  endPoint: joi.string(),
+  driverId: joi.number().integer().positive(),
+  userId: joi.number().integer().positive(),
+  status: joi.string().valid(...Object.values(OrderStatus)),
+  customerName: joi.string().pattern(UserValidationRule.NAME),
+  customerPhone: joi.string().pattern(UserValidationRule.PHONE),
```

Comment:
maybe not all of these fields can be updated in order ? for example userId/driverId/startPoint/endPoint ? Im not really sure that this data can be updated when the order is created already